include/roboteam_utils/Section.h

```diff
@@ -3,41 +3,74 @@
 #include "Vector2.h"
```

Comment:
I think the entire file is unused (and it's duplicate with LineSegment, I think).